Description: Remove documentation parts that do not apply to Debian
Author: Christian Perrier <bubulle@debian.org>
Forwarded: not-needed

Index: samba-3.4.6/docs/manpages/swat.8
===================================================================
--- samba-3.4.6.orig/docs/manpages/swat.8
+++ samba-3.4.6/docs/manpages/swat.8
@@ -261,85 +261,6 @@ will be appended (e\&.g\&. log\&.smbclie
 .RS 4
 Print a summary of command line options\&.
 .RE
-.SH "INSTALLATION"
-.PP
-Swat is included as binary package with most distributions\&. The package manager in this case takes care of the installation and configuration\&. This section is only for those who have compiled swat from scratch\&.
-.PP
-After you compile SWAT you need to run
-\FCmake install \F[]
-to install the
-\FCswat\F[]
-binary and the various help files and images\&. A default install would put these in:
-.sp
-.RS 4
-.ie n \{\
-\h'-04'\(bu\h'+03'\c
-.\}
-.el \{\
-.sp -1
-.IP \(bu 2.3
-.\}
-/usr/local/samba/sbin/swat
-.RE
-.sp
-.RS 4
-.ie n \{\
-\h'-04'\(bu\h'+03'\c
-.\}
-.el \{\
-.sp -1
-.IP \(bu 2.3
-.\}
-/usr/local/samba/swat/images/*
-.RE
-.sp
-.RS 4
-.ie n \{\
-\h'-04'\(bu\h'+03'\c
-.\}
-.el \{\
-.sp -1
-.IP \(bu 2.3
-.\}
-/usr/local/samba/swat/help/*
-.sp
-.RE
-.SS "Inetd Installation"
-.PP
-You need to edit your
-\FC/etc/inetd\&.conf \F[]
-and
-\FC/etc/services\F[]
-to enable SWAT to be launched via
-\FCinetd\F[]\&.
-.PP
-In
-\FC/etc/services\F[]
-you need to add a line like this:
-.PP
-\FCswat 901/tcp\F[]
-.PP
-Note for NIS/YP and LDAP users \- you may need to rebuild the NIS service maps rather than alter your local
-\FC /etc/services\F[]
-file\&.
-.PP
-the choice of port number isn\'t really important except that it should be less than 1024 and not currently used (using a number above 1024 presents an obscure security hole depending on the implementation details of your
-\FCinetd\F[]
-daemon)\&.
-.PP
-In
-\FC/etc/inetd\&.conf\F[]
-you should add a line like this:
-.PP
-\FCswat stream tcp nowait\&.400 root /usr/local/samba/sbin/swat swat\F[]
-.PP
-Once you have edited
-\FC/etc/services\F[]
-and
-\FC/etc/inetd\&.conf\F[]
-you need to send a HUP signal to inetd\&. To do this use
-\FCkill \-1 PID \F[]
-where PID is the process ID of the inetd daemon\&.
 .SH "LAUNCHING"
 .PP
 To launch SWAT just run your favorite web browser and point it at "http://localhost:901/"\&.
@@ -357,14 +278,11 @@ This file must contain suitable startup 
 This file must contain a mapping of service name (e\&.g\&., swat) to service port (e\&.g\&., 901) and protocol type (e\&.g\&., tcp)\&.
 .RE
 .PP
-\FC/usr/local/samba/lib/smb\&.conf\F[]
+\FC/etc/samba/smb\&.conf\F[]
 .RS 4
 This is the default location of the
 \fBsmb.conf\fR(5)
-server configuration file that swat edits\&. Other common places that systems install this file are
-\FC /usr/samba/lib/smb\&.conf\F[]
-and
-\FC/etc/smb\&.conf \F[]\&. This file describes all the services the server is to make available to clients\&.
+server configuration file that swat edits\&. This file describes all the services the server is to make available to clients\&.
 .RE
 .SH "WARNINGS"
 .PP
Index: samba-3.4.6/docs/manpages/nmbd.8
===================================================================
--- samba-3.4.6.orig/docs/manpages/nmbd.8
+++ samba-3.4.6/docs/manpages/nmbd.8
@@ -265,10 +265,7 @@ used by
 to answer any name queries\&. Adding a line to this file affects name NetBIOS resolution from this host
 \fIONLY\fR\&.
 .sp
-The default path to this file is compiled into Samba as part of the build process\&. Common defaults are
-\FC/usr/local/samba/lib/lmhosts\F[],
-\FC/usr/samba/lib/lmhosts\F[]
-or
+The default path to this file is
 \FC/etc/samba/lmhosts\F[]\&. See the
 \fBlmhosts\fR(5)
 man page for details on the contents of this file\&.
@@ -337,7 +334,7 @@ If running the server via the meta\-daem
 \FCinetd\F[], this file must contain a mapping of service name (e\&.g\&., netbios\-ssn) to service port (e\&.g\&., 139) and protocol type (e\&.g\&., tcp)\&.
 .RE
 .PP
-\FC/usr/local/samba/lib/smb\&.conf\F[]
+\FC/etc/samba/smb\&.conf\F[]
 .RS 4
 This is the default location of the
 \fBsmb.conf\fR(5)
@@ -388,7 +385,7 @@ is to send it a SIGTERM (\-15) signal an
 will accept SIGHUP, which will cause it to dump out its namelists into the file
 \FCnamelist\&.debug \F[]
 in the
-\FC/usr/local/samba/var/locks\F[]
+\FC/var/run/samba\F[]
 directory (or the
 \FCvar/locks\F[]
 directory configured under wherever Samba was configured to install itself)\&. This will also cause
Index: samba-3.4.6/docs/manpages/smbd.8
===================================================================
--- samba-3.4.6.orig/docs/manpages/smbd.8
+++ samba-3.4.6/docs/manpages/smbd.8
@@ -319,14 +319,11 @@ If running the server via the meta\-daem
 \FCinetd\F[], this file must contain a mapping of service name (e\&.g\&., netbios\-ssn) to service port (e\&.g\&., 139) and protocol type (e\&.g\&., tcp)\&.
 .RE
 .PP
-\FC/usr/local/samba/lib/smb\&.conf\F[]
+\FC/etc/samba/smb\&.conf\F[]
 .RS 4
 This is the default location of the
 \fBsmb.conf\fR(5)
-server configuration file\&. Other common places that systems install this file are
-\FC/usr/samba/lib/smb\&.conf\F[]
-and
-\FC/etc/samba/smb\&.conf\F[]\&.
+server configuration file\&.
 .sp
 This file describes all the services the server is to make available to clients\&. See
 \fBsmb.conf\fR(5)
Index: samba-3.4.6/docs/manpages/lmhosts.5
===================================================================
--- samba-3.4.6.orig/docs/manpages/lmhosts.5
+++ samba-3.4.6/docs/manpages/lmhosts.5
@@ -257,10 +257,8 @@ file is in the same directory as the
 file\&.
 .SH "FILES"
 .PP
-lmhosts is loaded from the configuration directory\&. This is usually
-\FC/etc/samba\F[]
-or
-\FC/usr/local/samba/lib\F[]\&.
+lmhosts is loaded from the configuration directory\&. This is
+/etc/samba\&.
 .SH "VERSION"
 .PP
 This man page is correct for version 3 of the Samba suite\&.
Index: samba-3.4.6/docs/manpages/tdbbackup.8
===================================================================
--- samba-3.4.6.orig/docs/manpages/tdbbackup.8
+++ samba-3.4.6/docs/manpages/tdbbackup.8
@@ -227,7 +227,7 @@ Samba \&.tdb files are stored in various
 .\}
 
 \FCsecrets\&.tdb\F[]
-\- usual location is in the /usr/local/samba/private directory, or on some systems in /etc/samba\&.
+\- usual location is in the /var/lib/samba directory\&.
 .RE
 .sp
 .RS 4
@@ -240,7 +240,7 @@ Samba \&.tdb files are stored in various
 .\}
 
 \FCpassdb\&.tdb\F[]
-\- usual location is in the /usr/local/samba/private directory, or on some systems in /etc/samba\&.
+\- usual location is in the /var/lib/samba directory\&.
 .RE
 .sp
 .RS 4
@@ -253,7 +253,7 @@ Samba \&.tdb files are stored in various
 .\}
 
 \FC*\&.tdb\F[]
-located in the /usr/local/samba/var directory or on some systems in the /var/cache or /var/lib/samba directories\&.
+located in the /var/lib/samba and /var/run/samba directories\&.
 .SH "VERSION"
 .PP
 This man page is correct for version 3 of the Samba suite\&.
